scripts/generate-openclaw-config.py

```
@@ -733,7 +732,7 @@ def _placeholder(channel: str, env_key: str) -> str:
     for _plugin_id, _provider_keys in _bundled_provider_plugins.items():
         if provider_key not in _provider_keys:
             plugin_entries[_plugin_id] = {"enabled": False}
-
+    if "discord" in _ch_cfg: plugin_entries["discord"] = {"enabled": True}
```

Ruff (0.15.14)

[error] Multiple statements on one line (colon)

(E701)